site/src/theme.ts

```diff
@@ -5,6 +5,8 @@ const createBreakpoint = (value: number) => {
     };
 };
 
+export const gridTotalColumns = 24;
+
 export const theme = {
```

Comment:
Should this just be inside the theme, if it's already in the theme file?

```suggestion
export const theme = {
    gridTotalColumns: 24,
```

Reply:
It makes things more complicated because the variable is used outside the theme context. Then you have to manually import the whole theme from this file.